.github/ISSUE_TEMPLATE/FEATURE_REQUEST.md

```diff
@@ -0,0 +1,25 @@
+---
+name: 💡 Feature Request
+about: Report a Backend bug on Kubeflow Pipelines
+title: "[feature request]: [Description]"
```

Comment:
Shall we make the prefix more concise?
e.g `[feature]`

Reply:
Make sense! Updated.